# this string allows to change the path to the Haar classifier model
# string
# data_directory: path_to_data -> please change this parameter in the launch file

# this flag enables or disables the face recognition step
# bool
enable_face_recognition: true

# this list specifies the persons who shall be recognized by the face recognizer, comment out this line to avoid any specification at startup
# string[] e.g. ['albert', 'bernhard', 'caesar']
#identification_labels_to_recognize: ['albert', 'bernhard', 'caesar']

# desired width and height of the Eigenfaces (=eigenvectors)
# int
eigenface_size: 100

# number of eigenvectors per person to identify -> controls the total number of eigenvectors
# int
eigenvectors_per_person: 5

# Threshold to facespace
# double
threshold_facespace: 13000

# Threshold to detect unknown faces
# double
threshold_unknown: 2500

# metric for nearest neighbor search in face space: 0 = Euklidean, 1 = Mahalanobis, 2 = Mahalanobis Cosine
# int
metric: 0

# enables some debug outputs
# bool
debug: false
